nrf52_blenano2 tmp112 sensor sample build failed - redefined I2C

Hi,
The issue https://github.com/zephyrproject-rtos/zephyr/issues/8391
Is still not fixed...
Even if I am removing recent commit https://github.com/zephyrproject-rtos/zephyr/commit/f5ddc99c4fa8589ce6915c991ce136ab0cadd82b

```
[5/93] Building C object zephyr/arch/arm/core/cortex_m/CMakeFiles/arch__arm__core__cortex_m.dir/prep_c.c.obj
[6/93] Building C object zephyr/arch/arm/core/cortex_m/CMakeFiles/arch__arm__core__cortex_m.dir/scb.c.obj
FAILED: ccache /home/axel/zephyr-sdk/sysroots/x86_64-pokysdk-linux/usr/bin/arm-zephyr-eabi/arm-zephyr-eabi-gcc -DBUILD_VE
RSION=v1.12.0-286-g92778e4 -DKERNEL -DNRF52832_XXAA -D_FORTIFY_SOURCE=2 -D__ZEPHYR__=1 -I/home/axel/zephyr-current/kernel
/include -I/home/axel/zephyr-current/arch/arm/include -I/home/axel/zephyr-current/arch/arm/soc/nordic_nrf/nrf52 -I/home/a
xel/zephyr-current/arch/arm/soc/nordic_nrf/nrf52/include -I/home/axel/zephyr-current/arch/arm/soc/nordic_nrf/include -I/h
ome/axel/zephyr-current/boards/arm/nrf52_blenano2 -I/home/axel/zephyr-current/include -I/home/axel/zephyr-current/include
/drivers -Izephyr/include/generated -isystem /home/axel/zephyr-sdk/sysroots/x86_64-pokysdk-linux/usr/lib/arm-zephyr-eabi/
gcc/arm-zephyr-eabi/6.2.0/include -isystem /home/axel/zephyr-sdk/sysroots/x86_64-pokysdk-linux/usr/lib/arm-zephyr-eabi/gc
c/arm-zephyr-eabi/6.2.0/include-fixed -I/home/axel/zephyr-current/lib/libc/minimal/include -I/home/axel/zephyr-current/ex
t/hal/cmsis/Include -I/home/axel/zephyr-current/ext/hal/nordic/nrfx -I/home/axel/zephyr-current/ext/hal/nordic/nrfx/drive
rs/include -I/home/axel/zephyr-current/ext/hal/nordic/nrfx/hal -I/home/axel/zephyr-current/ext/hal/nordic/nrfx/mdk -I/hom
e/axel/zephyr-current/ext/hal/nordic/. -I/home/axel/zephyr-current/subsys/bluetooth -Os -g -Wall -Wformat -Wformat-securi
ty -Wno-format-zero-length -imacros /home/axel/nrf52/tmp112/build/zephyr/include/generated/autoconf.h -ffreestanding -Wno
-main --sysroot /home/axel/zephyr-sdk/sysroots/armv5-zephyr-eabi/usr -mthumb -mcpu=cortex-m4 -std=c99 -fno-asynchronous-u
nwind-tables -fno-pie -fno-pic -fno-strict-overflow -Wno-pointer-sign -Wno-unused-but-set-variable -fno-reorder-functions
 -fno-defer-pop -Werror=implicit-int -ffunction-sections -fdata-sections -mabi=aapcs -march=armv7e-m -MD -MT zephyr/CMake
Files/zephyr.dir/misc/generated/configs.c.obj -MF zephyr/CMakeFiles/zephyr.dir/misc/generated/configs.c.obj.d -o zephyr/C
MakeFiles/zephyr.dir/misc/generated/configs.c.obj   -c zephyr/misc/generated/configs.c
zephyr/misc/generated/configs.c: In function ‘_ConfigAbsSyms’:
zephyr/misc/generated/configs.c:194:145: error: expected expression before ‘)’ token
 GEN_ABSOLUTE_SYM(CONFIG_I2C_0_IRQ_PRI, );
                                                                                                                         
                        ^
In file included from /home/axel/zephyr-current/include/arch/arm/arch.h:20:0,
                 from /home/axel/zephyr-current/include/arch/cpu.h:15,
                 from /home/axel/zephyr-current/arch/arm/core/cortex_m/reset.S:17:
zephyr/include/generated/generated_dts_board.h:127:0: warning: "CONFIG_I2C_0_NAME" redefined
 #define CONFIG_I2C_0_NAME  NORDIC_NRF5_I2C_40003000_LABEL
 
In file included from <command-line>:0:0:
/home/axel/nrf52/tmp112/build/zephyr/include/generated/autoconf.h:187:0: note: this is the location of the previous defin
ition
 #define CONFIG_I2C_0_NAME "I2C_0"
 
In file included from /home/axel/zephyr-current/include/arch/arm/arch.h:20:0,
                 from /home/axel/zephyr-current/include/arch/cpu.h:15,
                 from /home/axel/zephyr-current/arch/arm/core/cortex_m/reset.S:17:
zephyr/include/generated/generated_dts_board.h:129:0: warning: "CONFIG_I2C_0_IRQ_PRI" redefined
 #define CONFIG_I2C_0_IRQ_PRI  NORDIC_NRF5_I2C_40003000_IRQ_0_PRIORITY
 
In file included from <command-line>:0:0:
/home/axel/nrf52/tmp112/build/zephyr/include/generated/autoconf.h:189:0: note: this is the location of the previous defin
ition
 #define CONFIG_I2C_0_IRQ_PRI
 
In file included from /home/axel/zephyr-current/include/arch/arm/arch.h:20:0,

```

--- a/boards/arm/bbc_microbit/Kconfig.board
+++ b/boards/arm/bbc_microbit/Kconfig.board
@@ -6,4 +6,5 @@
 
 config BOARD_BBC_MICROBIT
 	bool "BBC MICRO:BIT"
+	select HAS_DTS_I2C
 	depends on SOC_NRF51822_QFAA

--- a/boards/arm/bbc_microbit/bbc_microbit.dts
+++ b/boards/arm/bbc_microbit/bbc_microbit.dts
@@ -21,13 +21,13 @@
 &uart0 {
 	status = "ok";
 	current-speed = <115200>;
-	sda-pin = <30>;
-	scl-pin = <0>;
 };
 
 &i2c0 {
 	status = "ok";
 	clock-frequency = <I2C_BITRATE_FAST>;
+	sda-pin = <30>;
+	scl-pin = <0>;
 };
 
 &flash0 {

--- a/boards/arm/nrf52_blenano2/Kconfig.board
+++ b/boards/arm/nrf52_blenano2/Kconfig.board
@@ -8,4 +8,5 @@
 config BOARD_NRF52_BLENANO2
 	bool "nRF52 BLENANO2"
 	select HAS_DTS_GPIO
+	select HAS_DTS_I2C
 	depends on SOC_NRF52832_QFAA

--- a/boards/arm/nrf52_blenano2/nrf52_blenano2.dts
+++ b/boards/arm/nrf52_blenano2/nrf52_blenano2.dts
@@ -27,14 +27,6 @@
 	status ="ok";
 };
 
-&i2c0 {
-	status = "ok";
-};
-
-&i2c1 {
-	status = "ok";
-};
-
 &uart0 {
 	compatible = "nordic,nrf-uart";
 	current-speed = <115200>;
@@ -42,6 +34,7 @@
 };
 
 &i2c0 {
+	status = "ok";
 	sda-pin = <28>;
 	scl-pin = <2>;
 };

